src/java/org/apache/cassandra/tools/StandaloneScrubber.java

```diff
@@ -80,7 +81,7 @@ public static void main(String args[])
     {
         Options options = Options.parseArgs(args);
 
-        if (Boolean.getBoolean(Util.ALLOW_TOOL_REINIT_FOR_TEST))
+        if (TEST_UTIL_ALLOW_TOOL_REINIT_FOR_TEST.getBoolean())
```

Comment:
`Util.ALLOW_TOOL_REINIT_FOR_TEST` is actually not used anywhere so it might be removed. All access to that property goes via cassandra properties class.